Caboodle/Geolocation/Geolocation.ios.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CoreLocation;
+using Foundation;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Geolocation
+    {
+        internal static bool IsSupported
+            => CLLocationManager.LocationServicesEnabled;
+
+        static async Task<Location> PlatformLastKnownLocationAsync()
+        {
+            await Permissions.RequireAsync(PermissionType.LocationWhenInUse).ConfigureAwait(false);
```

Comment:
Can we remove the configureawait for now throughout. I am still worried about that stuff.